lib/puppet_x/chocolatey/chocolatey_common.rb

```diff
@@ -30,12 +30,12 @@ def chocolatey_command
           #   it installs
           chocoInstallPath = PuppetX::Chocolatey::ChocolateyInstall.install_path
 
-          chocopath =  (chocoInstallPath if (chocoInstallPath && file_exists?("#{chocoInstallPath}\\bin\\choco.exe"))) ||
-              ('C:\ProgramData\chocolatey' if file_exists?('C:\ProgramData\chocolatey\bin\choco.exe')) ||
-              ('C:\Chocolatey' if file_exists?('C:\Chocolatey\bin\choco.exe')) ||
+          chocopath =  (chocoInstallPath if (chocoInstallPath && file_exists?("#{chocoInstallPath}\\choco.exe"))) ||
```

Comment:
Would there ever be the case of `choco.exe` not existing but `bin\choco.exe` does